tests/resources/queue-event-prs.json

```diff
@@ -29,6 +29,16 @@
       "eventSource": "aws:sqs",
       "eventSourceARN": "arn:aws:sqs:eu-west-2:006106226016:cert-gen-q",
       "awsRegion": "eu-west-2"
+    },
+    {
+      "messageId": "98a8d895-429d-48d4-8ac6-3a430aab75cp",
+      "receiptHandle": "AQEBJcBvTRZ1W2LSaUJ0g0ELXlqA8WCL4zJxO63wu0YOVhx44xxxPhsnc+/Q9+1vOPYO+3HupEjXzGRSvfPY5rEEJkgCJe4/RQ+q2kU5LsmJEr1qE/CTdIYe5X/75XeMQ523KKpdNsD9tRhyvEpPpSu50byGbz7J0JyR6lu1E6Q4YuB4QNm+ev1obPMLdEt8RhgvIi/NfEfQf0L1r3TPi3wLho1R61PllPm27He8/1CjCnMyWBzgX+DCjJ7vyRXObMZ/MbhMBKbYpeTcejsKpYX//PPr1yvldp1YPC0wPKp+iqmWxoDDeHXbo8xYRFXDA8rnY5RfkwxxffH7o534vYn8FCZEtqybQuo7pumu6Ah9PsC05tP38syU71ltasljGIA35BgCdSO+9r5rTaBnbO9++Q==",
```

Comment:
Can you update the prs flag on the defect to true please, as there is only 1 defect and the test result is prs, if it was false the result would be fail.